routes/api/scoreboard.js

```diff
@@ -1,30 +1,45 @@
 const express = require('express');
 const debug = require('debug')('rgbctf-backend');
 const Team = require('../../models/team');
+const Challenge = require('../../models/challenge');
 
 const router = express.Router();
 
 router.get('/:index', (req, res) => {
   const index = +req.params.index;
-  if (index === undefined) {
+  if (index === undefined || Number.isNaN(index) || index === Infinity) {
     res.send({ success: false, err: 'bad index' });
     res.end();
     return;
   }
   Team.find({}, 'name points', {
     skip: index,
-    limit: index + 20,
+    limit: 100,
     sort: {
       points: 1,
     },
   }, (e, teams) => {
     if (e) {
-      debug(`login/user: err: ${e}`);
+      debug(`scoreboard/team: err: ${e}`);
       res.send({ success: false, err: 'internal error' });
+      res.end();
     } else {
-      res.send({ success: true, teams: teams.map(({ name, points }) => ({ name, points })) });
+      Challenge.find({}, 'points', (challE, challs) => {
+        if (challE) {
+          debug(`scoreboard/chall: err: ${challE}`);
+          res.send({ success: false, err: 'internal error' });
```

Comment:
i just pushed a pr which adds middleware, so you could call `next(createError(500, 'Internal Error'));`